.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -69,6 +69,8 @@ Zebra follows [semantic versioning](https://semver.org). Semantic versions look
 Choose a release level for `zebrad`. Release levels are based on user-visible changes from the changelog:
 - Mainnet Network Upgrades are `major` releases
 - significant new features, large changes, deprecations, and removals are `minor` releases
+    - any new, changed, or removed RPC methods or fields are a `minor` release
+    - any new, changed, or removed command-line arguments are a `minor` release
```

Comment:
@arya2 @gustavovalverde did you have any specific feedback or better wording about command-line changes?

Optional: Is there a way to summarise this better?

Reply:
Should new config fields be a minor release too?

Reply:
> Should new config fields be a minor release too?

Yep, usually configs are a new feature, a new ability to configure an existing feature, or another behaviour change.

I think the IP connection limit in 1.0.1 was an exception, because the behaviour did change, but due to a security fix. And the config was intended for developers and testers only.

Reply:
I added "configs" to my suggestions in https://github.com/ZcashFoundation/zebra/pull/7196#discussion_r1258979877